src/Azure.Functions.Cli/StaticResources/funcignore

```diff
@@ -3,4 +3,5 @@
 .git*
 .vscode
 local.settings.json
-test
\ No newline at end of file
+test
```

Comment:
Should we also include `.venv/` and `tests/` in here? Since this is the data structure we included in getting_started_python_function.md

Reply:
Ideally - `.venv` and `tests` should be ignored - and we need to figure out if they should be added to `.gitignore` along with `funcignore`.

Also - tests are currently not something we generate, and shouldn't add explicitly - folks might want that as a part of the tracked source. This needs clarification.